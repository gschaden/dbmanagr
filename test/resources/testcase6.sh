db-navigator.py -s bmw
