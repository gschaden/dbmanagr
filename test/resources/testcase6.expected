Id	Title	Subtitle	Autocomplete	
skiline@bmwqa.skiline.net/	skiline@bmwqa.skiline.net/	Connection	skiline@bmwqa.skiline.net/
postgres@bmwqa.skiline.net/skiline	postgres@bmwqa.skiline.net/skiline/	Connection	postgres@bmwqa.skiline.net/skiline/
postgres@bmwqa.skiline.net/	postgres@bmwqa.skiline.net/	Connection	postgres@bmwqa.skiline.net/
skiline@bmwqa.skiline.net/skiline	skiline@bmwqa.skiline.net/skiline/	Connection	skiline@bmwqa.skiline.net/skiline/

